wiki/Beatmap/Marathon/de.md

```diff
@@ -0,0 +1,15 @@
+---
+stub: true
+---
+
+# Marathon
+
+Der Begriff **Marathon** wird verwendet, um [Beatmaps](/wiki/Beatmap) mit einer Länge von mehr als fünf Minuten zu beschreiben. Diese Beatmaps sind oft [Kollaborationen](/wiki/Beatmap/Beatmap_collaborations), da das Mapping von längeren Songs für einen einzelnen Mapper anstrengend sein kann.
+
+## Geschichte
+
+Als Marathon-Beatmaps zuerst eingeführt wurden, mussten sie mindestens *sechs* Minuten lang sein und durften nur eine einzelne Schwierigkeitsstufe enthalten (mit der Ausnahme, dass jede Beatmap noch eine zusätzliche [osu!taiko](/wiki/Game_mode/osu!taiko)-Schwierigkeitsstufe beinhalten konnte).
+
+Marathon-Beatmaps mussten ihre einzelne Schwierigkeitsstufe außerdem "Marathon" nennen, um das damalige Beatmap-Submission-System zu umgehen und nicht in die [Ausstehend](/wiki/Beatmap/Category#work-in-progress-und-ausstehend)-Kategorie eingeordnet zu werden. Statt der [Ranked](/wiki/Beatmap/Category#ranked)-Kategorie wurden die Beatmaps dann in die [Approved](/wiki/Beatmap/Category#approved)-Kategorie eingeordnet, um zu signalisieren, dass es sich um Marathon-Beatmaps handelt. Später wurden neue Marathon-Beatmaps dann doch in die Ranked-Kategorie eingeordnet.
+
+Später wurde eingeführt, dass auch Beatmaps mit fünf Minuten an [Drain-Zeit](/wiki/Beatmap/Drain_time) als Marathon-Beatmaps zählen und dass [benutzerdefinierte Namen für Schwierigkeitsgrade](/wiki/Ranking_criteria/Difficulty_naming) verwendet werden können. Dieses System blieb bis 2019 gleich und wurde dann von den weniger strengen Regeln zu Schwierigkeits-Spreads ersetzt. <!-- TODO: Update the translation for "Schwierigkeits-Spreads" when the article for it exists -->
```

Comment:
```suggestion
Später wurde eingeführt, dass auch Beatmaps mit fünf Minuten an [Drain-Zeit](/wiki/Beatmap/Drain_time) als Marathon-Beatmaps zählen und dass [benutzerdefinierte Namen für Schwierigkeitsstufen](/wiki/Ranking_criteria/Difficulty_naming) verwendet werden können. Dieses System blieb bis 2019 gleich und wurde dann von den weniger strengen Regeln zu Schwierigkeits-Spreads ersetzt. <!-- TODO: Update the translation for "Schwierigkeits-Spreads" when the article for it exists -->
```

Reply:
My suggestion is to use `festgelegt` instead of `eingeführt`.